recipe/meta.yaml

```diff
@@ -1,81 +1,84 @@
 {% set name = "imagecodecs" %}
-{% set version = "2021.8.26" %}
+{% set version = "2023.1.23" %}
```

Comment:
I wonder if we could update to the latest version https://github.com/cgohlke/imagecodecs/releases/tag/v2023.7.10. because v2023.1.23 was released 6 months ago

Reply:
We could, but I noticed that they dropped `py3.8` for that version. Is that OK for us? I wanted to have a recent version with `py3.8` because it's in our range of `py` version. If it's not relevant I will overwrite this and go for the last version or the latest that support `py3.8` which is `2023.3.16` (there should be a PR already there but it's not updated)

Reply:
I will update this PR to `2023.3.16` with `py3.8`,

once this is merged and done, I will update #13 to [2023.7.10](https://github.com/cgohlke/imagecodecs/releases/tag/v2023.7.10)

Reply:
`2023.3.16` requires `libjpeg-turbo` which is not in conda `default`. It might be worth having it.

Reply:
I will not update this here,
will be done in PKG-2501